### DISCUSSION TITLE
How to set different time formats

### DISCUSSION THREAD

**Discussion by JamesZhutheThird** [george-gca/multi-language-al-folio#25] [Q&A] [answered]
### Have you checked that your feature request isn't already filed?

- [X] I read through [FAQ](https://github.com/george-gca/multi-language-al-folio/blob/master/FAQ.md) and searched through the past issues both in the [original repo](https://github.com/alshedivat/al-folio/issues) and [this one](https://github.com/george-gca/multi-language-al-folio/issues), none of which addressed my issue.
- [X] Yes, I have checked that this feature request isn't already filed.

### Description & Motivation

Many thanks for your multi-language support, this repo is fabulous. Is there a way to set different time formats for different languages? i.e. `YYYY年MM月DD日` for Chinese, and `DD Month, YYYY` for English?

**Comment by george-gca** [comment, marked as answer]
Currently this has to be done manually via [_includes/date_format.liquid](https://github.com/george-gca/multi-language-al-folio/blob/main/_includes/date_format.liquid). Check how this is used in [_pages/en-us/blog.md](https://github.com/george-gca/multi-language-al-folio/blob/main/_pages/en-us/blog.md).
